Misc/NEWS.d/next/Library/2020-11-23-23-42-08.bpo-42299.Fdn4Wf.rst

```diff
@@ -0,0 +1,2 @@
+Removed the ``formatter`` module, which was deprecated in 3.4.
+(Contributed by Dong-hee Na and and Terry J. Reedy in :issue:`33955`.)
```

Comment:
```suggestion
Patch by Dong-hee Na and and Terry J. Reedy.
```

The bpo number is part of the NEWS entry and will be generated in the Changelog: https://docs.python.org/dev/whatsnew/changelog.html#changelog